components/ArticleReply/index.js

```diff
@@ -0,0 +1,242 @@
+      user: articleReplyAuthor,
+    } = articleReply;
+
+    const { type: replyType } = reply;
+
+    const replyAuthor = reply.user;
+    const user = replyAuthor || articleReplyAuthor;
+
+    const classes = useStyles({ replyType });
+
+    const renderFooter = useCallback(() => {
+      const copyText =
+        typeof window !== 'undefined'
+          ? `${TYPE_NAME[reply.type]} \n【${t`Reason`}】${(
+              reply.text || ''
+            ).trim()}\n↓${t`Details`}↓\n${
+              window.location.href
+            }\n↓${t`Reference`}↓\n${reply.reference}`
+          : '';
+
+      return (
+        <Box component="footer" display="flex" py={2}>
+          {showFeedback && (
+            <ReplyFeedback
+              articleId={articleId}
+              replyId={replyId}
+              positiveFeedbackCount={positiveFeedbackCount}
+              negativeFeedbackCount={negativeFeedbackCount}
+              feedbacks={feedbacks}
+              ownVote={ownVote}
+              reply={reply}
+              className={classes.feedbacks}
+            />
+          )}
+          <ReplyShare copyText={copyText} />
+        </Box>
+      );
+    }, [reply, articleReply]);
+
+    const renderAuthor = useCallback(() => {
+      const articleReplyAuthorName =
+        (
+          <EditorName
+            key="editor"
+            editorName={articleReplyAuthor?.name}
+            editorLevel={articleReplyAuthor?.level}
+          />
+        ) || t`Someone`;
+
+      const originalAuthorElem = (
+        <EditorName
+          key="editorName"
+          editorName={replyAuthor?.name}
+          editorLevel={replyAuthor?.level}
+        />
+      );
+      const originalAuthorsReply = (
+        <Link key="originalReply" href="/reply/[id]" as={`/reply/${reply.id}`}>
+          <a>{jt`${originalAuthorElem}'s reply`}</a>
+        </Link>
+      );
+
+      if (replyAuthor?.name && articleReplyAuthor?.id !== replyAuthor?.id) {
+        return (
+          <span key="editor">
+            {jt`${articleReplyAuthorName} uses ${originalAuthorsReply} to`}
+          </span>
+        );
+      }
+
+      return articleReplyAuthorName;
+    }, [articleReplyAuthor, replyAuthor]);
+
+    const renderReference = useCallback(() => {
+      if (replyType === 'NOT_ARTICLE') return null;
+
+      const reference = reply.reference;
+      return (
+        <section className={classes.root}>
+          <h3>
+            {replyType === 'OPINIONATED' ? t`Different opinion` : t`Reference`}
+          </h3>
+          {reference
+            ? nl2br(linkify(reference))
+            : `⚠️️ ${t`There is no reference for this reply. Its truthfulness may be doubtful.`}`}
+
+          <Hyperlinks
+            hyperlinks={reply.hyperlinks}
+            pollingType="replies"
+            pollingId={replyId}
+          />
+        </section>
+      );
+    }, [replyType, reply]);
+
+    const authorElem = renderAuthor();
+
+    return (
+      <li className={classes.root}>
```

Comment:
1. Seems that this line does not appear? Currently there is no separation between replies, which is a bit hard to read. ![image](https://user-images.githubusercontent.com/108608/82017417-b0f03280-96b5-11ea-9b82-f19ee8656924.png)
2. According to [mockup](https://www.figma.com/file/zpD45j8nqDB2XfA6m2QskO/Cofacts-website?node-id=681%3A0) the horizontal line between replies should be solid lines.